client/src/Components/WelcomingPage/WelcomingScreen.tsx

```diff
@@ -0,0 +1,121 @@
+import {  Image, ActivityIndicator , TouchableOpacity, Alert } from 'react-native'
+import {useNavigation , NavigationProp} from "@react-navigation/native";
+import CustomButton from '../../CustomComponents/CustomButton';
+import CustomText from '../../CustomComponents/CustomText';
+import CustomView from '../../CustomComponents/CustomView';
+import { RootStackParamList } from '../../types/navigation';
+import getThemeStore from '../../stores/themeStore';
+import { darkMode, lightMode } from '../colors/colors';
+import { observer } from 'mobx-react-lite';
+import { useEffect } from 'react';
+import { mmkv } from '../SettingsPage/Settings';
+
+const welcomingImage = require("../../../../assets/welcome.png");
+
+
+
+
+const WelcomingScreen : React.FC= observer(() => {
+    const navigation = useNavigation<NavigationProp<RootStackParamList>>();
+    const theme = getThemeStore().isDarkThemeEnabled.get() ? darkMode : lightMode ;
+    useEffect(() => {
+        try {
+          const storedTheme = mmkv.getBoolean('themeEnabled');
+          console.log('Retrieved storedTheme:', storedTheme);
+          if (storedTheme !== undefined) {
+            getThemeStore().setThemeEnabled(storedTheme);
+          } else {
+            console.log('No stored theme found, defaulting to false');
+          }
+        } catch (error) {
+          console.error('Error retrieving themeEnabled from MMKV:', error);
+          Alert.alert('Error', 'Failed to retrieve theme setting.');
+        }
+      }, []);
+ 
+  return (
+    <CustomView
+    
+        style={{ 
+            display : 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+            backgroundColor: theme.backGroundColor,
+            height: "100%"
+        }}
+    >
+        <Image style={{
+            width: 400,
+            height: 250,
+            borderRadius: 950
+            }} 
+            source={welcomingImage} />
+        <CustomView style={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+            }}>
+            <CustomText
+                style={{
+                    color:theme.fontColor
+                }}
+                fontSize={30}
+                fontWeight='bold'
+            >Welcome to the news reader app</CustomText>
+            <CustomText
+                style={{
+                    paddingTop: 30,
+                    color: theme.fontColor
+                }}
+                fontSize={15}
+                fontWeight='400'
+            >We're excited to introduce you our news paper reader app.</CustomText>
+            <ActivityIndicator
+            style={{
+                marginTop: 20
+            }}
+                size={100}
+                color="#77E4C8"
```

Comment:
we don't use hex , we sue colors from theme

Reply:
DONE ✅